app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsFragmentPresenter.kt

```diff
@@ -161,6 +181,7 @@ class AdministratorControlsFragmentPresenter @Inject constructor(
     return when (selectedFragment) {
       PROFILE_LIST_FRAGMENT -> 1
       APP_VERSION_FRAGMENT -> 3
+      PROFILE_AND_DEVICE_ID_FRAGMENT -> 5
```

Comment:
TODO: nit cleanup, but move indexes & maybe multipane settings to enum to simplify this & above.